examples/typescript/passkey/src/App.tsx

```diff
@@ -0,0 +1,218 @@
+import { AccountAddress, Aptos, AptosConfig, Network, Secp256r1PublicKey, postAptosFaucet } from "@aptos-labs/ts-sdk";
+import { useState } from "react";
+import "./App.css";
+
+const COIN_STORE = "0x1::coin::CoinStore<0x1::aptos_coin::AptosCoin>";
+
+function App() {
+  const [credentialId, setCredentialId] = useState<string | null>(window.localStorage.getItem("credentialId"));
+  const [publicKey, setPublicKey] = useState<string | null>(window.localStorage.getItem("publicKey"));
+  const [recipientAddress, setRecipientAddress] = useState<string | null>(null);
+  const [sendAmount, setSendAmount] = useState<number>(0);
+  const [faucetIsLoading, setFaucetIsLoading] = useState<boolean>(false);
+  const [passkeyAddr, setPasskeyAddr] = useState<string | null>(null);
+  const [currentNetwork, setCurrentNetwork] = useState<Network>(Network.DEVNET);
+
+  const config = new AptosConfig({ network: currentNetwork });
+  const aptos = new Aptos(config);
+
+  const rpName = window.location.origin;
+  const rpID = window.location.hostname;
+  console.log(rpName, rpID);
+
+  /**
+   * Prints the balance of an account
+   * @param aptos
+   * @param name
+   * @param address
+   * @returns {Promise<*>}
+   *
+   */
+  const balance = async (aptos: Aptos, name: string, address: AccountAddress) => {
```

Comment:
could we keep this example in a node env - same as all the other examples on the SDK? https://github.com/aptos-labs/aptos-ts-sdk/blob/main/examples/typescript/keyless.ts

Reply:
I will have a try, but it maybe hard. Passkey (webauthn) requires browser API for interactive authentication process.